src/test/java/com/atlassian/bitbucket/jenkins/internal/util/TestUtils.java

```diff
@@ -28,6 +30,18 @@ public static String readFileToString(String relativeFilename) {
         }
     }
 
+    public static void copyFileToPath(String relativeFilename, String path) {
```

Comment:
this method is only used from the checkout in the `TestSCM` and that usage should probably not be there? So maybe remove this method as well?